sage_modabvar/abvar.py

```diff
@@ -646,15 +646,18 @@ def elliptic_curve(self):
             raise RuntimeError("Elliptic curve not found" +
                                " in installed database")
 
-        sturm = ModularSymbols(N, 2).sturm_bound()
         a = lambda p: f.modular_symbols(1).eigenvalue(p, 'a')
```

Comment:
> This is cached.

Computing `ModularSymbols(N, 2).sturm_bound()` isn't hard -- it's a trivial arithmetic function of the weight and level. What is hard is "computing up to the sturm bound for f".